index.html

```diff
@@ -233,5 +233,78 @@ <h1>About me</h1>
 				</ul>
 			</div>
 		</section>
+		<section id="form-section">
+			<div id="form-text" class="form-text">
+				<p class="text">
+					I'm always interested in hearing about new projects, so if you'd like
+					to chat please get in touch.
+				</p>
+			</div>
+			<form
+				action="https://formspree.io/f/form"
+				id="form"
+				method="post"
+				class="form"
+			>
+				<ul class="form-container">
+					<li class="form_item">
+						<input
+							type="text"
+							id="name"
+							placeholder="Full Name"
+							required
+							maxlength="30"
+						/>
+					</li>
+					<li class="form_item">
+						<input
+							type="email"
+							id="email"
+							required
+							placeholder="Email Address"
+						/>
+					</li>
+					<li>
+						<textarea name="text" id="text" required maxlength="500">
+Enter text here</textarea
+						>
```

Comment:
- [ ] Kindly make some changes to the form's `textarea` field. You can add the `placeholder` attribute to the `textarea` element and then place the message text there. The placeholder attribute is there to give a guide text to a user on what is to be entered in an input field.
- Hint 
```mdx
<textarea name="text" id="text" required maxlength="500" placeholder="Enter text here"></textarea>
```

- [ ] Kindly work on the styles for the icons below the form. Currently, you are not following the design guidelines on the Figma template given. Please fix the styles by removing the blue background of the icons.

**Your design**
![image](https://user-images.githubusercontent.com/50941074/166829174-449d05e0-a0f1-4a0f-8f5b-eca627d75b08.png)

**The Figma Design**
![image](https://user-images.githubusercontent.com/50941074/166829249-641d4f7a-8059-49a4-9b8f-223b10b9b53c.png)